_sass/base/_markdown.scss

```diff
@@ -0,0 +1,92 @@
+.markdown {
+  padding-bottom: 60px;
+  
+  // Text Blocks
+  p {
+    margin: 10px 0;
+    font-weight: 300;
+
+    @media (min-width: $tablet) {
+      margin: 20px 0;
+    }
+  }
+  
+
+  img {
+    width: 100%;
+  }
+
+  // List Items
+  li {
+    line-height: 24px;
+  }
+
+  // Lists
+  ol,
+  ul {
+    list-style: initial;
+    list-style-position: inside;
+    color: $black;
```

Comment:
Is there a reason why list items are black while the rest of the text is in text colour (dark grey)? It looks a bit odd/jumpy.